Makefile

```diff
@@ -405,16 +405,16 @@ $(OBJDIR)/%.o : $(CURDIR)/%.cu | $$(@D)/.DIR
 	$(call quiet,NVCC) $(CPPFLAGS) $(NVCCFLAGS) -c -o $@ $(abspath $<)
 
 $(OBJDIR)/% : tests/%.c | $$(@D)/.DIR
-	$(call quiet,LINK.c) -o $@ $(abspath $<) $(CEED_LIBS) $(LDLIBS)
+	$(call quiet,LINK.c) $(CEED_LDFLAGS) -o $@ $(abspath $<) $(CEED_LIBS) $(LDLIBS)
```

Comment:
In some places we use `LDFLAGS`, in others `CEED_LDFLAGS`, and in some places both. It is probably best to unify the usage to one of the two, unless there is a reason to have both.

Reply:
This is like `CEED_LIBS`; they are extra flags needed when linking `-lceed`. It's really so our examples can link with `-dynamic` on Crays that override compiler defaults to insist on static linking. It shouldn't need to be touched anywhere else.

Reply:
I'd be happy if you have suggestions for cleaner ways to do this, but it is not a property of the compiler that can be obtained with `$(CC) --version`. Since many (especially newer) Crays don't inject this behavior, I don't know a simple way to test for it.

Reply:
OK, I see. Since these two variables serve different purposes then there is no problem.